screenpy_requests/actions/__init__.py

```diff
@@ -23,11 +23,11 @@ def generate_send_method_class(method: str) -> Type[APIMethodAction]:
     """
 
     class SendMETHODRequest:
-        "Will be programmatically replaced."
+        """Will be programmatically replaced."""
```

Comment:
Oh yeah also i had this fix here for months. Haha.

Reply:
I figure when we add ruff to the project, all these would be updated anyway.